mappings/net/minecraft/client/model/Model.mapping

```diff
@@ -4,9 +4,16 @@ CLASS net/minecraft/class_3879 net/minecraft/client/model/Model
 	FIELD field_21343 layerFactory Ljava/util/function/Function;
 	METHOD <init> (Ljava/util/function/Function;)V
 		ARG 1 layerFactory
+	METHOD accept (Ljava/lang/Object;)V
+		ARG 1 part
 	METHOD method_23500 getLayer (Lnet/minecraft/class_2960;)Lnet/minecraft/class_1921;
 		ARG 1 texture
 	METHOD method_2828 render (Lnet/minecraft/class_4587;Lnet/minecraft/class_4588;IIFFFF)V
-		ARG 5 r
-		ARG 6 g
-		ARG 7 b
+		ARG 1 matrix
+		ARG 2 vertices
+		ARG 3 light
+		ARG 4 overlay
+		ARG 5 red
+		ARG 6 green
+		ARG 7 blue
+		ARG 8 alpha
```

Comment:
Already mapped in my PR.

Reply:
If yours is merged first I'll fix the conflicts.